portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,46 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+/* Shows about text when the "About" button is pressed. */
+function showAbout () {
+    const aboutText = "I'm a rising junior at UCLA, majoring in Linguistics and Computer Science (that's one major)! In my free time, I enjoy reading and playing the piano. \n\n Recently, I've started learning to cook (I'm going to need this for my college apartment next year) and playing Stardew Valley (turns out video games are a great way to stay connected to friends during this time)!";
```

Comment:
you could also consider this being part of the html instead of JS by putting it in a hidden DIV and showing it when the link is pressed instead of hardcoding this text in the method

might be easier to read when looking at the HTML file

Reply:
I like that idea, I've changed it